Makefile

```diff
@@ -19,71 +19,60 @@ else
 $(error "ARCH must be set to one of: x86, x86_64")
 endif
 
-# set default docker build image name
-BUILD_IMAGE_NAME ?= otelopscol-build
-
-OTELCOL_BINARY = google-cloud-metrics-agent_$(GOOS)_$(GOARCH)$(EXTENSION)
-
 ALL_SRC := $(shell find . -name '*.go' -type f | sort)
 ALL_DOC := $(shell find . \( -name "*.md" -o -name "*.yaml" \) -type f | sort)
 GIT_SHA := $(shell git rev-parse --short HEAD)
 
 BUILD_INFO_IMPORT_PATH := github.com/GoogleCloudPlatform/opentelemetry-operations-collector/internal/version
 BUILD_X1 := -X $(BUILD_INFO_IMPORT_PATH).GitHash=$(GIT_SHA)
 BUILD_X2 := -X $(BUILD_INFO_IMPORT_PATH).Version=$(PKG_VERSION)
+ifdef JMX_HASH
+JMX_RECEIVER_IMPORT_PATH := github.com/open-telemetry/opentelemetry-collector-contrib/receiver/jmxreceiver
+BUILD_X_JMX := -X $(JMX_RECEIVER_IMPORT_PATH).MetricsGathererHash=$(JMX_HASH)
+LD_FLAGS := -ldflags "${BUILD_X1} ${BUILD_X2} ${BUILD_X_JMX}"
+else
 LD_FLAGS := -ldflags "${BUILD_X1} ${BUILD_X2}"
+endif
 
 TOOLS_DIR := internal/tools
 
 .EXPORT_ALL_VARIABLES:
 
-.DEFAULT_GOAL := presubmit
-
-# --------------------------
-#  Install Tools
-# --------------------------
-
-.PHONY: install-tools
-install-tools:
-	cd $(TOOLS_DIR) && go install github.com/client9/misspell/cmd/misspell
-	cd $(TOOLS_DIR) && go install github.com/golangci/golangci-lint/cmd/golangci-lint
-	cd $(TOOLS_DIR) && go install github.com/google/addlicense
-	cd $(TOOLS_DIR) && go install github.com/google/googet/goopack
-	cd $(TOOLS_DIR) && go install github.com/pavius/impi/cmd/impi
-	cd $(TOOLS_DIR) && go install github.com/open-telemetry/opentelemetry-collector-contrib/cmd/mdatagen@v0.67.0
+.DEFAULT_GOAL := ci
 
 # --------------------------
-#  Helper Commands
+#  Helpers 
 # --------------------------
 
 .PHONY: update-components
 update-components:
 	grep -o github.com/open-telemetry/opentelemetry-collector-contrib/[[:lower:]]*/[[:lower:]]* go.mod | xargs -I '{}' go get {}
 	go mod tidy
+	cd $(TOOLS_DIR) && \
+		go get -u github.com/open-telemetry/opentelemetry-collector-contrib/cmd/mdatagen && \
+		go mod tidy
```

Comment:
If this command fails, it would be hard to figure out whether `go get` or `go mod tidy` failed. Keeping them as two separate commands would allow `make` to report the failure appropriately…

Reply:
Doing a `cd` requires that they all be one command right? I thought running a `cd` as a separate command in a make target won't actually change the whole directory context.

Reply:
I separated the update and tidy into two commands as suggested. For collapsing the tidy into the update loop, after looking at it I realize that the old `update-components` make target needs to be re-done anyway. It doesn't catch all opentelemetry dependencies right now. So I will probably be rewriting a chunk of that in another PR anyway.